### ISSUE TITLE
Stabilize clusters formed by BERTopic

### ISSUE THREAD

**Issue by sumedha05** [MaartenGr/BERTopic#321]
How do we finetune hyperparameters like learning rate, so as to reach to a point when we start receiving stable clusters after every model run.

**Comment by MaartenGr** [comment]
I am not exactly sure I understand your question. To which model do you refer when you talk about learning rate? Also, BERTopic does not train in epochs, it needs only a single pass (if you can call it that) to learn the topic representations.

**Comment by MaartenGr** [comment]
I am going to close this for now due to inactivity. If, however, you are still experiencing this issue, please let me know! I'll make sure to reopen it and see if we can discuss this further.
